### ISSUE TITLE
How to write the label file?

### ISSUE THREAD

**Issue by arunpatro** [zalandoresearch/disentangling_conditional_gans#1]
Hi, 

I couldn't figure out from the Readme.md and 
https://github.com/zalandoresearch/disentangling_conditional_gans/blob/03d5101ba63ed223d310e42584545bb1d53077e0/dataset.py#L89

about how the label file should be written as. From reading the paper, I understand that your Disentangling Conditional GAN perfoms color regression rather than categorical classification. 

From https://github.com/zalandoresearch/disentangling_conditional_gans/blob/03d5101ba63ed223d310e42584545bb1d53077e0/dataset_tool.py#L636, I figure that labels is a pickle dump of a dictionary. Are the keys of this dictionary names of the images and values of this dictionary numpy arrays of RGB values?

**Comment by gokyildirim** [comment]
Yes, in the paper, we perform color regression. The dictionary structure is as you have described, the keys are the file name of an image, and the values are the average RGB color of the associated object in the image. I will create an updated branch soon with an example toy dataset, I hope it will be clearer for you as well.

**Comment by arunpatro** [comment]
Could you comment on the timeline for this toy implementation?

**Comment by gokyildirim** [comment]
@arunpatro could you check the new branch "toydata", I hope it is clearer now.
